ansible_collections/arista/avd/plugins/plugin_utils/schema/avdtodocumentationschemaconverter.py

```diff
@@ -399,6 +399,36 @@ def description(self, schema: dict, indentation: str, table: str):
             return "<br>".join(descriptions)
         return None
 
+    def deprecation(self, schema: dict) -> tuple[str, str]:
```

Comment:
Please rebase, since this should be merged already.

Reply:
Have rebased now but i can still see this diff. No idea how to fix it.

Reply:
Just remove this whole function from this file. This PR should rely on the existing code (see the function `get_deprecation` at the top of this file, which is where the code moved to)

Reply:
Ok thanks, removed now!